ui/src/app/views/workspace-edit/component.html

```diff
@@ -14,11 +14,35 @@ <h2 class="page-header">Clone workspace "{{oldWorkspaceName}}"</h2>
       </ng-container>
       <div class="required-text">(Required)</div>
     </div>
-    <div class="name-area required-field" [class.validation-error]="nameNotEntered">
+    <div class="name-area required-field" [class.validation-error]="nameNotEntered"
+          style="display: inline-block; vertical-align: middle;">
       <input type="text" class="input name" name="workspace-name" autofocus
              [(ngModel)]="workspace.name" [disabled]="!hasPermission"
              placeholder="Workspace Name">
     </div>
+    <clr-dropdown style="display: inline-block; vertical-align: middle;">
+      <div class="tooltip">
+        <div class="tooltip-content-wrapper">
+          <button [disabled]="mode == Mode.Edit"
+                  class="btn btn-dropdown cdr-version-dropdown" type="button" clrDropdownTrigger>
+            {{selectedCdrName}}
+            <clr-icon shape="caret down"></clr-icon>
+          </button>
+          <div *ngIf="mode == Mode.Edit" class="tooltip-text">
+            To use a different dataset version, clone or create a new workspace.
+          </div>
+        </div>
+      </div>
+      <clr-dropdown-menu *clrIfOpen>
```

Comment:
What CDR version is getting selected by default? (It should be the default returned in the CDR versions endpoint...)

We should possibly have a message that appears such that when you pick an older CDR version, we warn you that it's only OK to do this if continuing with an existing research purpose or reproducing someone else's work.

Reply:
The default CDR version is selected by default.

Filed RW-1327 to track warning the user about the policy. I don't think we have the details/language right now.